src/redux/reducers/trendsStateByUser.js

```diff
@@ -47,8 +49,17 @@ const trendsStateByUser = (state = {}, action) => {
       return update(
         state,
         { [userId]: {
-          [FOCUSED_SLICE]: { $set: { slice, position } },
-          [FOCUSED_KEYS]: { $set: focusedKeys },
+          [FOCUSED_CBG_SLICE]: { $set: { slice, position } },
+          [FOCUSED_CBG_KEYS]: { $set: focusedKeys },
+        } }
+      );
+    }
+    case actionTypes.FOCUS_TRENDS_SMBG_RANGE_AVG: {
+      const { rangeAvgData: data, rangeAvgPosition: position, userId } = action.payload;
+      return update(
+        state,
+        { [userId]: {
+          [FOCUSED_SMBG_AVG]: { $set: { data, position } },
```

Comment:
OK, I think the components are:
- containers/trends/CBGSlicesAnimationContainer
- components/trends/cbg/FocusedCBGSliceTime
- component/trends/cbg/FocusedCBGSliceHTMLLabels

No blip code changes after all - the latter two components are *rendered* via blip but obviously the code for them is in viz.